tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -262,11 +263,31 @@ final ProcessRunnerResult _dryRunResult = ProcessRunnerResult(
   pid: 0, // pid,
 );
 
+bool _isFlag(String arg) {
+  return !arg.contains('=') && !arg.contains(' ') && !arg.startsWith('--');
+}
+
+(String, bool) _extractRawFlag(String flagArgument) {
+  assert(flagArgument.startsWith('--'), 'Must be a valid flag argument.');
+  var rawFlag = flagArgument.substring(2);
+  var flagValue = true;
+  if (rawFlag.startsWith('no-')) {
+    rawFlag = rawFlag.substring(3);
+    flagValue = false;
+  }
+  return (rawFlag, flagValue);
+}
```

Comment:
Looks like these were moved to another file, and need to be deleted here.

Reply:
Done.